core-web/libs/edit-content/src/lib/services/dot-edit-content.service.ts

```diff
@@ -54,6 +61,96 @@ export class DotEditContentService {
      * The type of the emitted contentlet is determined by the generic type parameter.
      */
     saveContentlet<T>(data: { [key: string]: string }): Observable<T> {
-        return this.dotWorkflowActionsFireService.saveContentlet(data);
+        return this.#dotWorkflowActionsFireService.saveContentlet(data);
+    }
+
+    /**
+     * Get data for site/folder field and tranform into TreeNode
+     * @returns files
+     */
```

Comment:
We normally use this extension to write jsdoc style documentation https://marketplace.visualstudio.com/items?itemName=oouo-diogo-perdigao.docthis